index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>African-Marketplace</title>
```

Comment:
I like that you have the Title Updated for the browser tab and possibly open graph api can pick it up if the url is shared on twitter of Facebook etc.

Radical.   🚀